README.md

```
@@ -80,25 +103,28 @@ uv run pytest          # 動作確認 (69 tests)
 }
 ```
 
-`uv` のフルパスは `which uv` で取得（PATH を継承しない環境があるため絶対パス推奨）。
+`uv` のフルパスは `which uv` で取得 (PATH を継承しない環境があるため絶対パス推奨)。
 
-### 3. 最初の概念を作る
+### 4. 最初の概念を作る
 
 Claude にこう話しかけるだけ:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)